designsafe/apps/rapid/views.py

```diff
@@ -11,6 +13,8 @@
 from django.contrib.auth.decorators import login_required, user_passes_test
 from django.contrib.auth import get_user_model, models
 from django.db.models import Q
+from tapipy.tapis import Tapis
```

Comment:
@Beulah-Karrolla , can you add the the changes to pyproject.toml and poetry.lock for the installed tapipy.

Reply:
addressed.